src/app/pages/organizations/developers/developer/join.jsx

```diff
@@ -22,10 +24,47 @@ import { useFetchDevelopers, usePutJoinDevelopers } from 'api/developer';
 import { ChplActionBar } from 'components/action-bar';
 import { ChplTextField } from 'components/util';
 import { getAngularService } from 'services/angular-react-helper';
+import ClearIcon from '@material-ui/icons/Clear';
+import AddIcon from '@material-ui/icons/Add';
+import { theme } from 'themes';
 
 const useStyles = makeStyles({
   headingPadding: {
```

Comment:
Can we change this class name? Now that's it's not just padding it should have a "reasonable" name. Honestly, it should have before, but we I missed it earlier